packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -282,6 +296,60 @@ export default class CashierStore extends BaseStore {
         }
     }
 
+    @computed
+    is_cashier_locked() {
+        if (!this.root_store.client.account_status) return false;
+        const { status } = this.root_store.client.account_status;
+
+        return status.some(status_name => status_name === 'cashier_locked');
+    }
+
+    @computed
+    is_deposit_locked() {
+        const {
+            is_authentication_needed,
+            is_tnc_needed,
+            is_financial_account,
+            is_financial_information_incomplete,
+            is_trading_experience_incomplete,
+            account_status,
+        } = this.root_store.client;
+
+        if (!account_status) return false;
+        const need_financial_assessment =
+            is_financial_account && (is_financial_information_incomplete || is_trading_experience_incomplete);
+
+        // is_authentication_needed return integer instead of boolean, it will make the proptypes failed
+        return !!is_authentication_needed || is_tnc_needed || need_financial_assessment;
+    }
+    @computed
+    is_transfer_locked() {
+        const {
+            is_financial_account,
+            is_financial_information_incomplete,
+            is_trading_experience_incomplete,
+            account_status,
+        } = this.root_store.client;
+
+        if (!account_status) return false;
+
+        const need_financial_assessment =
+            is_financial_account && (is_financial_information_incomplete || is_trading_experience_incomplete);
+
+        return need_financial_assessment;
+    }
+    @computed
```

Comment:
Nit: can we add spacing around this computed like the others?

```suggestion
    }

    @computed
    is_transfer_locked() {
        const {
            is_financial_account,
            is_financial_information_incomplete,
            is_trading_experience_incomplete,
            account_status,
        } = this.root_store.client;

        if (!account_status) return false;

        const need_financial_assessment =
            is_financial_account && (is_financial_information_incomplete || is_trading_experience_incomplete);

        return need_financial_assessment;
    }

    @computed
```